### PR TITLE
update core package

### PR DESCRIPTION
**Pull request by JacqueGM**
fix mutual connections display

### CODE REVIEW

**Comment by mikozet** [comment]
It looks like there are some conflicts...
could you reslove?

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "webpack-dev-server": "^3.11.3"
   },
   "dependencies": {
-    "@circles/core": "^4.6.0",
+    "@circles/core": "^4.7.1-14-g13446762",
```
Comment (mikozet) [inline package.json]: not here but in conversation tab it shows there are conflicts...

**Review by mikozet — DISMISSED** [review]

**Review by juanenrisley — APPROVED** [review]
👍🏿

**Comment by JacqueGM** [comment]
Related tickets https://github.com/CirclesUBI/circles-core/issues/224

**Pull request merged**